qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -174,13 +179,26 @@ def write_local_config(self):
             export TEST_DIR={}
             export SCRATCH_DEV={}
             export SCRATCH_MNT={}
-            export CEPHFS_MOUNT_OPTIONS="-o name=admin,secret={}"
+            export CEPHFS_MOUNT_OPTIONS="-o {}name=admin,secret={}"
             ''').format(self.test_dev, self.test_dirs_mount_path, self.scratch_dev,
-                        self.scratch_dirs_mount_path, self.get_admin_key())
+                        self.scratch_dirs_mount_path, _options, self.get_admin_key())
 
         self.mount_a.client_remote.write_file(join(self.xfstests_repo_path, 'local.config'),
                                               xfstests_config_contents, sudo=True)
 
+    def write_ceph_exclude(self):
+        from os.path import join
+        from textwrap import dedent
```

Comment:
Since we are using these imports in multiple functions, it'll be better to import once at the top of file than import in each function using it.

Reply:
sure.